Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -232,7 +258,14 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
   case ACMD_SENDOPCOND:
     // Sends host capacity support information (HCS) and asks the accessed card to send
     // its operating condition register (OCR) content
-    Memory::Write_U32(0x80ff8000, _BufferOut);
+    {
+      // Never leave idle state if the card is not supported by the protocol
```

Comment:
Dunno, but this comment still feels off to me. "Never leave idle" == "set Status to Initialized"? Comment ignorable if this is the case though.